cmd/neofs-cli/modules/util/convert_eacl.go

```diff
@@ -58,7 +58,7 @@ func convertEACLTable(cmd *cobra.Command, _ []string) {
 func prettyPrintJSON(cmd *cobra.Command, data []byte) {
```

Comment:
BTW, can this be replaced with `common.PrettyPrintJSON()`?

Reply:
Done.